src/filefinder.cpp

```diff
@@ -191,30 +195,49 @@ namespace {
 					}
 				}
 			}
-			return name;
+			return std::string();
 		}
 		return file_it->second;
 	}
 
 	std::string FindFile(const std::string &dir, const std::string& name, const char* exts[]) {
+		std::string rtp_name;
+
 		const std::shared_ptr<FileFinder::DirectoryTree> tree = FileFinder::GetDirectoryTree();
 		std::string const ret = FindFile(*tree, dir, name, exts);
 		if (!ret.empty()) { return ret; }
 
-		const std::string& rtp_name = translate_rtp(dir, name);
+		// Try RTP if enabled and available
+		if (!disable_rtp)
+			rtp_name = translate_rtp(dir, name);
+
+		if (!rtp_name.empty()) {
+			// RPG_RT will even load RTP files when the game disables it
+			if(disable_rtp_warnings && !warning_broken_rtp_game_shown) {
+				warning_broken_rtp_game_shown = true;
+				Output::Warning("This game claims it does not need the RTP, but actually uses files from it!");
+			}
 
-		for(search_path_list::const_iterator i = search_paths.begin(); i != search_paths.end(); ++i) {
-			if (! *i) { continue; }
+			for(search_path_list::const_iterator i = search_paths.begin(); i != search_paths.end(); ++i) {
```

Comment:
nit: space after for (yes, it was in the original, but it's a good chance)

Reply:
please don't fix whitespacing in existing code.
This makes rebasing even worse